sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_phone_numbers_client.py

```diff
@@ -219,35 +258,43 @@ def begin_update_phone_number_capabilities(
         :type sms: str or ~azure.communication.phonenumbers.PhoneNumberCapabilityType
         :param calling: Capability value for calling.
         :type calling: str or ~azure.communication.phonenumbers.PhoneNumberCapabilityType
-        :keyword str continuation_token: A continuation token to restart a poller from a saved state.
+        :keyword continuation_token: A continuation token to restart a poller from a saved state.
+        :paramtype continuation_token: str or None
         :keyword polling: Pass in True if you'd like the LROBasePolling polling method,
             False for no polling, or your own initialized polling object for a personal polling strategy.
         :paramtype polling: bool or ~azure.core.polling.PollingMethod
-        :keyword int polling_interval: Default waiting time (seconds) between two polls
+        :keyword polling_interval: Default waiting time (seconds) between two polls
             for LRO operations if no Retry-After header is present.
+        :paramtype polling_interval: int
         :returns: A poller to wait on the update operation.
         :rtype: ~azure.core.polling.LROPoller[~azure.communication.phonenumbers.PurchasedPhoneNumber]
         """
 
         capabilities_request = PhoneNumberCapabilitiesRequest(
             calling=calling, sms=sms)
 
-        polling_interval = kwargs.pop(
-            "polling_interval", _DEFAULT_POLLING_INTERVAL_IN_SECONDS)
-
         if not phone_number:
             raise ValueError("phone_number can't be empty")
 
         poller = self._phone_number_client.phone_numbers.begin_update_capabilities(
-            phone_number, body=capabilities_request, polling_interval=polling_interval, **kwargs
+            phone_number,
+            body=capabilities_request,
+            polling_interval=polling_interval,
+            continuation_token=continuation_token,
+            polling=polling,
+            **kwargs
         )
 
         result_properties = poller.result().additional_properties
```

Comment:
Can we open an issue to investigate this code?
We should not be calling `poller.result()` inside the method - as this will block the whole method until the polling operation has comleted. This also appears to be inconsistent with the async implementation.

Reply:
Created user story tp address in future PR